### PR TITLE
bump provider, flutter_map, latlong2 to latest version

### PR DESCRIPTION
**Pull request by Harishwarrior**
Fix for issue #47

### CODE REVIEW

**Comment by Harishwarrior** [comment]
@rorystephenson can you check this please?

**Comment by aytunch** [comment]
This is needed for map cluster plugin to be updated for flutter_map 2.0.0 too. Can this please be merged?

File: example/pubspec.yaml [hunk 1]
```diff
@@ -10,15 +10,15 @@ dependencies:
   flutter:
     sdk: flutter
 
-  latlong2: ^0.8.0
-  flutter_map: ^1.0.0
+  latlong2: ^0.8.1
+  flutter_map: ^2.0.0
```
Comment (pablojimpas) [inline example/pubspec.yaml:14]: Is it possible to use `>=1.0.0 <3.0.0` for backwards compatibility?

File: pubspec.yaml [hunk 2]
```diff
@@ -11,9 +11,9 @@ dependencies:
     sdk: flutter
 
   animated_stack_widget: ^0.0.4
-  flutter_map: ^1.0.0
+  flutter_map: ^2.0.0
```
Comment (pablojimpas) [inline pubspec.yaml:14]: Same here

Comment (aytunch) [reply pubspec.yaml:14]: flutter_map 2.1.1 is the latest version btw

Comment (pablojimpas) [reply pubspec.yaml:14]: > flutter_map 2.1.1 is the latest version btw

That is within the range that I suggested above

Comment (Harishwarrior) [reply pubspec.yaml:14]: I don't think that's a solution. It's more of a workaround.

**Comment by rorystephenson** [comment]
Sorry for the delay! I've published 3.2.0 which fixes this (I would have merged this PR but @pablojimpas's suggested version range seemed like a better solution than pinning to v2.

**Pull request closed without merging**